edition-nft/contracts/AllDaySeasonal.cdc

```diff
@@ -59,17 +63,17 @@ pub contract AllDaySeasonal: NonFungibleToken {
     // totalSupply
     // The total number of NFTs that in circulation.
     //
-    pub var totalSupply:        UInt64
+    access(all) var totalSupply:        UInt64
```

Comment:
Should we remove this field now that burning is not decrementing it anymore?

Reply:
good point

Reply:
deleted